packages/liveblocks-core/src/room.ts

```diff
@@ -353,9 +358,23 @@ export type Room<
      * room.subscribe("history", ({ canUndo, canRedo }) => {
      *   // Do something
      * });
-     *
      */
     (type: "history", listener: Callback<HistoryEvent>): () => void;
+
+    /**
+     * Subscribe to pending storage modifications updates.
+     *
+     * @returns Unsubscribe function.
+     *
+     * @example
+     * room.subscribe("pending-storage-modifications", ({ hasPendingStorageModifications }) => {
+     *   // Do something
+     * });
+     */
+    (
+      type: "pending-storage-modifications",
```

Comment:
We call these changes "storage updates" everywhere else. Should we stick with that nomenclature, i.e. rename this to `pending-storage-updates`? Also, because the meaning of this event is captured in a boolean value (has or has-not), should we reflect that in the name? For example: add a "has" or "is" to the name, like `has-pending-storage-updates`?

Going a step further even, could we rename this event to `is-synchronized` instead? To me, it's about avoiding double-negation in concepts, compare the "happy paths" `has-pending-storage-updates? false!` vs `is-synchronized? true!`. We're mostly interested in acting on the event when the system is in-sync again, after all.

Reply:
Just adding my $0.02 here now, because you only get one free shot when picking new names in an API.

Reply:
I like `is-synchronized`! It's way better. Do you think we should add the word storage to make it clear we're talking about the storage only? `is-storage-synchronized`?

Or maybe, we should go a step further and have a non boolean value? `storage-status` so we can support 'synchronized' / `synchronizing` / `offline`? 🤔

Reply:
Yeah, `is-storage-synchronized` might be better! Just to make it explicit _what_ is synced, agreed.

I have another alternative that I just thought of… if you look at the event sources we currently have defined, here:

https://github.com/liveblocks/liveblocks/blob/75419a17e5c667d61be041dd8c5901fb44998c6c/packages/liveblocks-core/src/room.ts#L599-L612

Look at the `storageDidLoad` event. This is the event without payload that only fires once during the lifetime of a room. What would you think of following this naming convention and change this setup to `storageDidSync` instead? When we use that name, we can remove the payload from the event, and just interpret each event as an indicator that storage just emptied its pending Ops buffer. Of course, there would not be a way to tell (event-wise) when it gets out of sync again, but I don't think we're interested in that event anyway?

Reply:
Updated the implementation. Far from ideal but the external API is as we discussed. Let's rework our internal a bit later.